src/parse_args.ts

```diff
@@ -65,7 +65,14 @@ program
     '--dry-run',
     "don't actually execute anything, report only registered journeys"
   )
-  .option('--journey-name <name>', 'only run the journey with the given name')
+  .option(
+    '--match <name>',
```

Comment:
`--grep`, `--match` i like both

Reply:
I feel like maybe we should match names and tags with `--match`, a flag like `--tags` or `--name` feels more like a limited version of `--match`...

Reply:
`--grep` has more historical meaning in my mind, and would be used to grep test names specifically.

Reply:
Hmmm, I like that idea. We can stick to match and pass the glob to do global name and tag matching. @andrewvc WDYT?

Reply:
I'm OK with matching both. I guess regexps would be for more precise name-only matching?

Reply:
Had a offline discussion, We will go with this. 
```
- match pattern that matches both name and tags
- tags pattern that matches only tags
```